src/main/java/dk/bec/polonez/reservationsystem/controller/FeatureController.java

```diff
@@ -25,12 +26,24 @@ public FeatureController(FeatureService featureService){
 
     @GetMapping("{id}")
     public Feature getFeatureById(@PathVariable long id) {
-        return featureService.getById(id);
+        return featureService.getFeatureById(id);
```

Comment:
**info** A previous (and shorter) name ```featureService.getById``` was readable because ```getById``` method was defined in ```featureService``` so that the context was obvious: the method was responsible for getting **feature** by id, not sth else (book/car/student/user/discount).
General remark: If you use a power of **context**, then your method/field/class names can be shorter.